### PR TITLE
update travis script to point to new location of openwhisk repo

### LINKED ISSUE #284: update travis script to use new location of openwhisk repo url

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #284

### CODE REVIEW

File: tools/travis/install_openwhisk.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ HOMEDIR="$(dirname "$TRAVIS_BUILD_DIR")"
 cd $HOMEDIR
 
 # OpenWhisk stuff
-git clone --depth 3 https://github.com/openwhisk/openwhisk.git
+git clone --depth 3 https://github.com/apache/incubator-openwhisk.git
```
Comment (mrutkows) [inline tools/travis/install_openwhisk.sh]: to preserve paths to an "/openwhisk" do we want to use:
git clone --depth 3 https://github.com/apache/incubator-openwhisk.git openwhisk

to not create an "incubator-openwhisk" directory which might cause other issues?

File: tools/travis/install_openwhisk.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ HOMEDIR="$(dirname "$TRAVIS_BUILD_DIR")"
...
 
 # Build script for Travis-CI.
 WHISKDIR="$HOMEDIR/openwhisk"
```
Comment (mrutkows) [inline tools/travis/install_openwhisk.sh]: if we clone incubator-openwhisk, will this directory still exist?

Comment (lzbj) [inline tools/travis/install_openwhisk.sh]: correct, we need to change to
WHISKDIR="$HOMEDIR/incubator-openwhisk"

**Comment by lzbj** [comment]
Close it as latest commit merged.

**Pull request closed without merging**